packages/types/package.json

```diff
@@ -48,7 +48,8 @@
   },
   "files": [
     "build",
-    "dist"
+    "dist",
+    "dist-ts3.8"
```

Comment:
I didn't want to change the naming of `dist` since that's what they used upstream